station/Classes/Dfu/DfuFlasher.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import iOSDFULibrary
+
+protocol DfuFlasherOutputProtocol: AnyObject {
+    func ruuviDfuDidUpdateProgress(percentage: Float)
+    func ruuviDfuDidUpdateLog(log: DfuLog)
+    func ruuviDfuDidFinish()
+    func ruuviDfuError(error: Error)
+}
+
+class DfuFlasher: NSObject {
+    private let queue = DispatchQueue(label: "DfuFlasher", qos: .userInteractive)
+    private var dfuServiceInitiator: DFUServiceInitiator!
+    private weak var output: DfuFlasherOutputProtocol?
+    private var firmware: DFUFirmware!
+    private var partsCompleted: Int = 0
+    private var currentFirmwarePartsCompleted: Int = 0
+
+    override init() {
+        super.init()
+        dfuServiceInitiator = DFUServiceInitiator(queue: queue,
```

Comment:
yes, we are using Swinject with force unwrapped properties for injection. 
But here we don't use injection, so we can go with more easy way? 
![2021-06-07_15-02-36](https://user-images.githubusercontent.com/722716/120998460-8d58c800-c7a1-11eb-8529-0010f57c69eb.png)

Reply:
```
private let dfuServiceInitiator: DFUServiceInitiator
    private weak var output: DfuFlasherOutputProtocol?
    private var firmware: DFUFirmware?
    private var partsCompleted: Int = 0
    private var currentFirmwarePartsCompleted: Int = 0
    private var dfuServiceController: DFUServiceController?

    override init() {
        self.dfuServiceInitiator = DFUServiceInitiator(
            queue: queue,
            delegateQueue: queue,
            progressQueue: queue,
            loggerQueue: queue
        )
        super.init()
    }
```